src/App.jsx

```
@@ -1,245 +1,326 @@
+const AuroraChat = createLazyComponent(() => import("@/pages/aurora-site/aurora-chat"));
+const CommunityInteractionPage = createLazyComponent(() => import("@/pages/aurora-site/community/community"));
+const TeacherDirectoryPage = createLazyComponent(() => import("@/pages/aurora-site/teacher-directory/teacher-directory"));
+const LeaderboardPage = createLazyComponent(() => import("@/pages/aurora-site/community/leaderboard"));
+const PublicProfile = createLazyComponent(() => import("@/pages/public-profile/public-profile"));
 
 // 🏛️ Grammar & Language
-import GrammarContent from "@/pages/aurora-site/grammar-content";
-import PresentSimpleCoursePage from "@/pages/aurora-site/present-simple-course";
-import SocialMediaCoursePage from "@/pages/aurora-site/social-media-course";
-import PastSimpleCoursePage from "@/pages/aurora-site/past-simple-course";
+const GrammarContent = createLazyComponent(() => import("@/pages/aurora-site/grammar-content"));
```

Biome (2.1.2)

[error] Shouldn't redeclare 'GrammarContent'. Consider to delete it or rename it.

'GrammarContent' is defined here:


(lint/suspicious/noRedeclare)